DP1.10058.001/Table.joining.md

```diff
@@ -1,6 +1,8 @@
 |Table 1|Table 2|Join by field(s)|
 |------------------------|------------------------|-------------------------------|
 div_1m2Data|div_10m2Data100m2Data|Direct join not recommended. See Standard calculations.
-div_geneticarchive|Any other table|Join not recommended. Data resolution does not match other tables.
+div_1m2Data|div_morphospecies|siteID and morphospeciesID
```

Comment:
Needs to be siteID, morphospeciesID to be read correctly by neonOS.

Reply:
I had neglected to add the new(ish) identificationHistory table.  With that, I removed the "no other tables" lines for the genetic archive and voucher